src/components/input/inputFileAvatar/inputFileAvatar.api.ts

```diff
@@ -0,0 +1,46 @@
+import type { IInputComponentProps } from '../inputContainer';
+
+export const SelectState = {
+    IDLE: 'idle',
+    SELECTING: 'selecting',
+    SUCCESS: 'success',
+    ERROR: 'error',
+} as const;
+
+export type SelectState = (typeof SelectState)[keyof typeof SelectState];
+
+export interface IInputFileAvatarProps
+    extends Omit<
+        IInputComponentProps,
+        | 'maxLength'
+        | 'onChange'
+        | 'variant'
+        | 'inputLength'
+        | 'wrapperClassName'
+        | 'invisible'
+        | 'alert'
```

Comment:
The component should handle the `alert` property eventually set by the parent components